### DISCUSSION TITLE
Open DTU über USB direk aus FritzBox speisen

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#1387] [Q&A]
Hallo,

ich betreibe eine OpenDTU über ein normales 5V-Steckernetzteil.
Nun habe ich der Standort der DTU verändert und sie befindet sich nahe der FritzBox.

Ich möchte nun das Steckernetzteil nicht mehr weiter benutzen, sondern die 5V-Spannung direkt aus einem freien USB-Port von der FritzBox nehmen.

Prinzipiell ist diese Spannungsversorgung ja kein Problem.

Ich weiß aber nicht, ob die im USB-Kabel ja weiterhin noch vorhandnen Datenleitungen zu Problemen führen können, in der DTU oder in der FritzBox (6591 cable).

Kann mir jemand hier eine Antwort geben?
In der DTU ist ein ESP32-D0WD-V3 verbaut.

Vielen Dank im Voraus und Grüße.

**Comment by Tiese** [comment]
Hallo Fokker-1,

in der Fritz!Box-Doku steht...

> Die FRITZ!Box versorgt USB-Geräte gemäß USB-3.0-Spezifikation mit insgesamt bis zu 900 mA. Bei höherem Strombedarf verbinden Sie die Geräte über einen USB-Hub mit eigener Stromversorgung (aktiver USB-Hub) mit der FRITZ!Box.

Schon aus diesem Grund würde ich es bei dem Netzteil belassen.

**Comment by Fokker-1** [comment]
Hallo Tiese,

danke für die Info.

Ich habe die Stromaufnahme meiner DTU an 5V mit 185 mA gemessen, dass sollte also kein Problem sein.

Mein Bedenken sind eher, ob die verbundenen Datenleitungen DTU - FritzBox hier "Unfug" anstellen könnten.

Grüße

**Reply by Tiese** [reply]
In der Anleitung von openDTU steht aber...

> Use a power suppy with 5 V and 1 A. The USB cable connected to your PC/Notebook may be powerful enough or may be not.

Folglich würde ich es nicht an der Fritz!Box anschließen!

Ob eine Datenverbindung zu Stande kommt, siehst du, wenn der ESP32 von der Fritz!Box erkannt wird.

**Reply by fkoehr** [reply]
Verwende doch einfach ein USB-**Lade**kabel, die sind explizit >>ohne<< Datenleitungen.
Die Leitungen würden hier nur unnötige Kosten verursachen, so kann ein Hersteller wenige sub-Cent sparen ;o) Das ist verfügbare Massenware ...
 
Musst halt nur beim Kauf genau in die Specs gucken, oder in den Stecker schielen, falls du noch ausrangierte USB-Kabel rumliegen hast ... LG

**Comment by Fokker-1** [comment]
Hallo Thiese,

danke für die Antwort.
Wie schon geschrieben: die Stromaufnahme ist 185 mA, also kein Problem.

Klar, die WLAN-Verbindung sieht man über der FritzBox.

Meine Frage war, ob ggf. die Datenverbindung auf dem **USB-Kabel** Probleme machen könnte.
Habe es jetzt getestet: keine Probleme.

**Reply by Omega13x** [reply]
Wenn die DTU auf Sendung geht, schießt der Strombedarf aber ordentlich in die Höhe. Dafür wäre mir die Fritzbox zu schade. Wenn du dann noch einen NRF24L01+ für die HM-Serie nutzt, wirst du dich über ständige Probleme im 2,4Ghz Wlan-Netz ärgern. Ist wie in der Kneipe, wenn viele Leute dicht beisammen sitzen und alle gleichzeitig reden.

**Reply by NintendoTito** [reply]
Bei mir hats wochenlang problemlos funktioniert. Muss aber auch gestehen, dass ich mir über mögliche Probleme oder Folgen keine Gedanken gemacht habe. Dachte mir eher so "ist halt USB, was soll passieren". Wie auch immer, funktioniert hat es.

**Reply by Fokker-1** [reply]
Hallo Omega13x
danke für die Antwort, aber so richtig verstanden habe ich es (noch) nicht.

Du schreibst:
_"Wenn die DTU auf Sendung geht, schießt der Strombedarf aber ordentlich in die Höhe. "
Welche Werte werden nach Deinen Erkenntnissen erreicht?

Du schreibst:
"Wenn du dann noch einen NRF24L01+ für die HM-Serie nutzt ..."
Was ist nach Deinen Erkenntnissen die Ursache für die Störungen durch diesen Funk-Chips bei 2,4GHz?

Ein nicht so schöner Nebeneffekt beim Speisen über USB aus der FritzBox ist, dass die OpenDTU immer mit Spannung versorgt wird. Das bedeutet, dass auch WLAN immer an ist in der Box, wenn man das Feature "WLAN ausschalten, sobald kein WLAN-Gerät mehr verbunden ist" verwenden will zur Energieersparnis. Die DTU sendet ja immer.

Ich habe aktuell daher die zeitgesteuerte WLAN-Abschaltung ohne dieses Feature gemacht.
Ich werde das aber noch ändern und in das USB-Kabel einen kleinen Schalter für die 5V einbauen.

Danke und Grüße

**Reply by io2345** [reply]
Ich möchte noch einen Schritt weitergehen und die OpenDTU-Platine im Fritzboxgehäuse unterbringen. Auf die Idee, die 5V von der USB-Buchse zu nutzen, bin ich durch diesen Artikel gekommen ;-) 
Werde berichten, ob das platzmäßig geht (falls ich es nicht vergesse)

**Comment by Slummi** [comment]
Nein, das USB-Kabel macht keine Probleme. Habe die DTU schon immer an der FritzBox hängen.

**Comment by io2345** [comment]
Geht und läuft schon. Fritzbox ist eine 7330, die als Repeater läuft:
![20231102_153654](https://github.com/tbnobody/OpenDTU/assets/111304560/c30ad43c-38ad-4fd7-9c47-dcd28a21f961)
Platz ist offenbar genug
![20231102_162538](https://github.com/tbnobody/OpenDTU/assets/111304560/2494c687-947c-4949-8284-5501584d15fe)
Modul "angeflanscht"
![20231102_194144](https://github.com/tbnobody/OpenDTU/assets/111304560/b822cdcf-bbf6-40c7-b792-ee08d6a38f53)
Direkt am seitlichen USB-Anschluss die 5V abgegriffen
![20231102_200650](https://github.com/tbnobody/OpenDTU/assets/111304560/27700026-459c-43ef-b639-cb086d8956f2)
Auf gute Heimwerkerart befestigt ;-) Auf die untere Abschirmung doppelseitiges Klebeband, damit ein transparentes Kunststoffblatt festgeklebt, auf dieses mit doppelseitigem Klebeband die DTU gepappt. Hinten die Sub-1G-Antenne befestigt
![20231102_200803](https://github.com/tbnobody/OpenDTU/assets/111304560/7b1fba7f-ed30-42d0-b271-e2cd3e4cf47f)
Die rote LED der DTU ist so hell, dass man sie sogar durch das rote Plastik der Box leuchten sieht...
![20231102_201422](https://github.com/tbnobody/OpenDTU/assets/111304560/3d51f1bc-471e-4a88-9eb5-934eec4be556)
Sieht doch schick aus, oder?

**Reply by fkoehr** [reply]
Das sieht echt chic und praktikabel aus, danke für die Idee :)

**Comment by Fokker-1** [comment]
Hallo iO2345,

interessante Lösung, Glückwunsch. Kann ich noch nicht machen, da noch Garantie auf der FB ist.

Zwischenzeitlich habe ich eine PV-Anlage installiert.
Das Auslesen eines IR-Lesekopfes am neuen, elektronischen Zähler habe ich realsiert (über Tasmota) und auch ins WLAN der FB eingebunden. 

Da ich im Zählerschrank keine Steckdose habe und auch grundsätzlich kein 5V-Steckernetzteil verwenden will, habe ich auch noch diese Versorgung - außer der openDTU - vom USB-Port der FB entnommen.

Dazu habe ich in ein kleines Kästchen eine fliegende Verdrahtung zur Verteilung der 5V vom USB auf die beiden Geräte gemacht. Jeder Zweig hat einen kleinen Schalter und eine LED zur Kontrolle. Ebenso ist ein Elko eingefügt und jeder Zweig getrennt über eine Mikro-Sicherung 0,5A abgesichert. Damit ist sicher gestellt, dass bei Störungen (z. B. Kurzschluss) auf einem Versorgungszweig keinerlei Rückwirkungen auf die FB möglich sind.

Die Kabellänge zum IR-Kopf beträgt etwa 4m, sie ist mit geschirmtem Kabel ausgeführt (LAN-Kabel). Die Schirmung ist überall durchverbunden.

Grüße
